### ISSUE TITLE
Revisit workflow for publishing centres from the global registry

### ISSUE THREAD

**Issue by amilan17** [wmo-im/wis2-topic-hierarchy#197]
There was a one time update from GR implemented last July. I *think there have been new centres added since then.  
https://github.com/wmo-im/global-registry/compare/d4147e4d2a47f41cd7b46aaf0af78587485ce930...b8fe70ee05fd1f5670d06b6c38a55d6c85b2745d

**Comment by amilan17** [comment]
@tomkralidis

**Comment by tomkralidis** [comment]
There was [initial work](https://github.com/wmo-im/global-registry/tree/test-ci-again) earlier in 2024 to develop a GitHub Action which would automagically generate the WTH centre-id CSV from the Global Registry and commit/push/publish.  Unfortunately GitHub Actions are not possible across repositories, when one repository is private (as is Global Registry).

@amilan17 I'm happy to convert the GitHub Action to a script that you can run in Global Registry if you wish.  Steps are per below:

```mermaid
sequenceDiagram
    participant WIS2 GlobalRegistry
    participant WMO
    participant WTH

    opt CSV generation
      WMO ->>+ WIS2 GlobalRegistry: run script to generate centre-id CSV
      WIS2 GlobalRegistry ->>+ WMO: return generated CSV
      WMO ->>+ WTH: commit generated CSV and push
    end
```

**Comment by amilan17** [comment]
@tomkralidis ok. let's to that.

**Comment by tomkralidis** [comment]
> @tomkralidis we should at least have the script run automatically whenever main is updated. The manual step would be to commit the generated CSV to WTH.

Workflow in https://github.com/wmo-im/global-registry/pull/50

**Comment by chenxiaoxia2019** [comment]
> Apparently, the Global Registry is so private, that I cannot even see it and therefore this discussion remains a mystery to me :-)

Yes, Global Registry is currently a private repo with the WIS2nodes local broken information.

**Comment by amilan17** [comment]
@tomkralidis I think we can close this now....
